hpat/hiframes/pd_series_ext.py

```diff
@@ -241,6 +241,7 @@ def __init__(self, dmm, fe_type):
 make_attribute_wrapper(SeriesType, 'name', '_name')
 
 
+from hpat.datatypes.hpat_pandas_series_functions import *
```

Comment:
I think we shouldn't move the import. @shssf please correct me if I'm wrong.

Reply:
Unfortunately this import is still needed. We will remove it in future. If you have better solution how to make new style functions - please let me know.

Reply:
I did in this way to overlap Array by Series: [#192](https://github.com/IntelPython/hpat/pull/192/files#diff-f13c21aac60fb8f7ae3c9527239d1e17R991-R999)